pkg/impl/libraptorq/swig/swig.swigcxx

```diff
@@ -1094,3 +1094,20 @@ public:
 
 %template(BytesEncoder) RFC6330__v1::Impl::Encoder<unsigned char *, unsigned char *>;
 %template(BytesDecoder) RFC6330__v1::Impl::Decoder<unsigned char *, unsigned char *>;
+
+%inline %{
```

Comment:
is it generated by swig?

Reply:
See above.  FWIW, none of these files have been generated by SWIG.  They are _fed_ into SWIG though.  SWIG-generated files are used internally when we run `go build` on this package.